lib/etcd/backup.go

```diff
@@ -44,6 +45,14 @@ func (b *BackupConfig) CheckAndSetDefaults() error {
 	if b.Writer == nil {
 		b.Writer = os.Stdout
 	}
+
+	// Etcd server might take longer than normal to respond to the large queries used for backup
+	// Coordinate library doesn't accept a 0 value, and will instead overwrite 0 to 1 second.
```

Comment:
This sounds like we should either fix this in coordinate to leave the timeout alone or ditch the library and set up etcd configuration manually - see no reason to work this around?

Reply:
No idea, coordinate largely predates me, so I have no idea what's using it, other than it was probably intentionally spun off into it's own library. 

I agree in general with just ditching it and setting up clients, but in this narrow case, where we've already ditched backup/restore based upgrades in 7.0, we don't really need etcd-backup anymore either, making this only to workaround problems on the older LTS releases.